src/core/MOM_PressureForce.F90

```diff
@@ -28,10 +28,10 @@ module MOM_PressureForce
 type, public :: PressureForce_CS ; private
   logical :: Analytic_FV_PGF !< If true, use the analytic finite volume form
                              !! (Adcroft et al., Ocean Mod. 2008) of the PGF.
-  logical :: blocked_AFV     !< If true, used the blocked version of the ANALYTIC_FV_PGF
+  logical :: blocked_FV      !< If true, used the blocked version of the ANALYTIC_FV_PGF
```

Comment:
blocked_FV is left over from an obsolete option and it not used anywhere in this module, so rather than being renamed, I think that it should simply be deleted.

Reply:
Addressed in ec0946c9c9d7e0f50a78451be67bc80ab8bd91dd